content/post/owasp-top10/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Will be Speaking at the Security Engineer MeetUp in Kyushu."
+date: 2022-03-02T12:37:33+09:00
+hero: /post/owasp-top10/top.png
+excerpt: "Introduction of OWASP Top10:2021"
+authors:
+  - Akitsugu Ito
+draft: false
+license: cc-by
```

Comment:
General:
The article is not a blog post but an announcement. It's probably too late now, but for the next time I think we should add a new [taxonomy](https://gohugo.io/content-management/taxonomies/) for events, talks, stuff like that and somehow visually differentiate them on the top page.